lib/plugins/aws/deploy-function.js

```diff
@@ -392,8 +392,14 @@ class AwsDeployFunction {
 
     if (!Object.keys(_.omit(params, 'FunctionName')).length) {
       if (this.options['update-config']) log.notice();
+      const noticeMessage = [
+        'Configuration did not change.',
+        ' Configuration update skipped.',
```

Comment:
Maybe lest's reword it to:

```
Function configuration did not change, and the update was skipped.
If you made changes to the service configuration and expected them to be deployed, it most likely means that they can only be applied with a full service deployment.
```

Reply:
Sure, will get back with the changes, thanks for the feedback.

Reply:
@medikoo I have incorporated the changes as requested.